src/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/commands.py

```diff
@@ -0,0 +1,139 @@
+        g.show_command('show', 'get')
+        g.command('list', 'list')
+        g.command('delete', 'delete')
+        g.custom_command('create', 'create_pool',
+                         client_factory=pools_mgmt_client_factory,
+                         doc_string_source='azure.mgmt.netapp.models#CapacityPool',
+                         exception_handler=netappfiles_exception_handler)
+        g.generic_update_command('update',
+                                 setter_name='update',
+                                 custom_func_name='patch_pool',
+                                 setter_arg_name='body',
+                                 doc_string_source='azure.mgmt.netapp.models#CapacityPool',
+                                 exception_handler=netappfiles_exception_handler)
+
+    with self.command_group('netappfiles volume', netappfiles_volumes_sdk) as g:
+        g.show_command('show', 'get')
+        g.command('list', 'list')
+        g.command('delete', 'delete')
+        g.custom_command('create', 'create_volume',
+                         client_factory=volumes_mgmt_client_factory,
+                         doc_string_source='azure.mgmt.netapp.models#Volume',
+                         exception_handler=netappfiles_exception_handler)
+        g.generic_update_command('update',
+                                 setter_name='update',
+                                 custom_func_name='patch_volume',
+                                 setter_arg_name='body',
+                                 doc_string_source='azure.mgmt.netapp.models#VolumePatch',
+                                 exception_handler=netappfiles_exception_handler)
+
+    with self.command_group('netappfiles volume export-policy', netappfiles_volumes_sdk) as g:
+        g.generic_update_command('add',
+                                 setter_name='update',
+                                 custom_func_name='add_export_policy_rule',
+                                 setter_arg_name='body',
+                                 doc_string_source='azure.mgmt.netapp.models#VolumePatch',
+                                 exception_handler=netappfiles_exception_handler)
+        g.custom_command('list', 'list_export_policy_rules',
+                         client_factory=volumes_mgmt_client_factory,
+                         doc_string_source='azure.mgmt.netapp.models#Volume',
+                         exception_handler=netappfiles_exception_handler)
+        g.generic_update_command('remove',
+                                 setter_name='update',
+                                 custom_func_name='remove_export_policy_rule',
+                                 setter_arg_name='body',
+                                 doc_string_source='azure.mgmt.netapp.models#VolumePatch',
+                                 exception_handler=netappfiles_exception_handler)
+
+    with self.command_group('netappfiles mount-target', netappfiles_mount_targets_sdk) as g:
+        g.command('list', 'list')
```

Comment:
Unless you plan on having additional "mount-target" commands, this should be `netappfiles list-mount-targets`.

Reply:
ok, will update because I see it is little effort to change and it is more likely that this will be removed rather than new ones added

Reply:
@tjprescott Ok, didn't work quite as expected. Currently the command.py contains a command_group and a command but this change removes the command group. Not sure how this should change

    _with self.command_group('netappfiles mount-target', netappfiles_mount_targets_sdk) as g:
         g.command('list', 'list')_